CMakeLists.txt

```diff
@@ -8,6 +8,6 @@ else()
 endif()
 conan_basic_setup()
 
-add_compile_options(-std=c++11)
```

Comment:
If this worked in cmake 2.8.12, I wouldn't update it. 
There are still some conan users that are using cmake 2.8.12, so it is good to try to keep as compatible with them as possible.